numpy/lib/tests/test_format.py

```diff
@@ -838,7 +838,11 @@ def test_large_file_support():
 
 @dec.slow
 def test_large_archive():
-    a = np.empty((2 ** 30, 2), dtype=np.uint8)
+    try:
+        a = np.empty((2**26, 2), dtype=np.uint8)
```

Comment:
Ah, a comment on the test intent would have been helpful.....

Indeed, needs to be skipped on 32-bit systems then. For the original size I get:

    In [1]: a = np.empty((2**30, 2), dtype=np.uint8)
    ---------------------------------------------------------------------------
    ValueError                                Traceback (most recent call last)
    <ipython-input-1-d8d46384be90> in <module>()
    ----> 1 a = np.empty((2**30, 2), dtype=np.uint8)

    ValueError: array is too big.

Will add the skip, but keep the ``except MemoryError`` - that's always good when trying to allocate such large arrays.